Zend/zend_variables.h

```diff
@@ -48,6 +48,15 @@ static zend_always_inline void i_zval_ptr_dtor(zval *zval_ptr)
 	}
 }
 
+static zend_always_inline void zend_array_ptr_dtor(zend_array *array)
```

Comment:
zend_array_release would be a more typical name. (zend_string_release, zend_object_release)

Reply:
Right. I went ahead and committed it separately: https://github.com/php/php-src/commit/718e55c3e045a3b786749e0fbedda7f0ab444907

Weirdly only spotted one existing use of the pattern, I thought it's more common.